skiko/build.gradle.kts

```diff
@@ -6,8 +6,9 @@ import org.jetbrains.kotlin.gradle.tasks.KotlinNativeCompile
 plugins {
     kotlin("multiplatform") version "1.5.31"
     `maven-publish`
+    signing
     id("org.gradle.crypto.checksum") version "1.1.0"
-    id("de.undercouch.download") version "4.1.1"
+    id("de.undercouch.download")
```

Comment:
The problem is that the download plugin is a dependency of a buildSrc module. So it leaks into the plugin classpath. In turn, Gradle prohibits specifying a version for a plugin, that's already present in a plugin classpath.
There are multiple ways to avoid leaking:
1. To shade & relocate `de.undercouch.download` into `:buildSrc:publishing`.
2. To rewrite downloading using okhttp.
I will do one of these later. Currently it does not seem like an important issue.